ui/src/app/core/initial-error/initial-error.component.ts

```diff
@@ -23,7 +23,7 @@ export class InitialErrorComponent {
         this.initialLoadFailure = true;
         const status = e.error.status || 'unknown';
         const title = e.error.title || 'Unknown error';
-        this.initialLoadErrorHeader = `${status}: ${title}`
+        this.errorMessage = `${status}: ${title}`
```

Comment:
Will also need to rename property at line 11

Reply:
:arrow_up:  this is why the UI test is failing